virttest/utils_net.py

```diff
@@ -1765,11 +1771,11 @@ def get_neigh_attch_interface(neigh_address):
     return get_neighbours_info(neigh_address)[neigh_address]["attach_if"]
```

Comment:
Do we need support `session` here? And would you help correct the typo in doc string: `wihch`?

Reply:
okay, currently it is not necessary but it is right way to add `session` support for `neigh_reachable()` and `get_neigh_attch_interface()` which depends on `get_neighbours_info()`. I will fix the typo too.